apps/exam/serializers.py

```diff
@@ -59,8 +60,11 @@ def get_answer_attempts(self, obj):
         for obj in obj.answerattempt_set.all()
     ]
 
+  def get_mode(self, obj):
```

Comment:
Not sure about this one (may be not necessary)

Reply:
It seems like it is necessary after all. Doesn't work with out it.